### ISSUE TITLE
Mintistall just crushes on Mint Victoria.

### ISSUE THREAD

**Issue by ivan-suhorukov** [linuxmint/mintinstall#400]
Hi! `mintinstall` crushes on my machine every time i start it, and it does not write anything in console. How can i figure out what is going wrong?

OS Version: Linux Mint 21.2 Victoria
Mintinstall Version: 8.2.8

```
$ mintinstall
MintInstall: Detected system architecture: 'x86_64'
```

Purging of `mintinstall` does not help.

Thanks!

**Comment by Quantum-mutnauQ** [comment]
Hello,
do you try 'sudo apt upgrade' to update the system?

**Comment by ivan-suhorukov** [comment]
> Hello, do you try 'sudo apt upgrade' to update the system?

Nope, it does not help.

**Comment by Quantum-mutnauQ** [comment]
you can do `python3.10 "/usr/lib/linuxmint/mintinstall/mintinstall.py"`
if it don't woke try to delete the cache `rm -r ~/.cache/mintinstall`

**Comment by Quantum-mutnauQ** [comment]
you can do `python3.10 "/usr/lib/linuxmint/mintinstall/mintinstall.py"`
if it don't woke try to delete the cache `rm -r ~/.cache/mintinstall`
if it than not woke can you make a short video?

**Comment by ivan-suhorukov** [comment]
No, that does not help eather, but then i run `python3.10 "/usr/lib/linuxmint/mintinstall/mintinstall.py"` it writes new error to console: `segmentation fault`. I make a video for you.

https://github.com/linuxmint/mintinstall/assets/141321266/076e0558-3835-4d71-8802-0b2640719732

**Comment by Quantum-mutnauQ** [comment]
Hello,
can you reubload the video it says the file is damaged.

**Comment by ivan-suhorukov** [comment]
Here it is:

https://github.com/linuxmint/mintinstall/assets/141321266/abb46d0b-d490-476c-97a3-310904416087

**Comment by mtwebster** [comment]
Please try running it this way:
```
G_MESSAGES_DEBUG=all DEBUG=1 mintinstall
```

And paste the output.

Thanks

**Comment by ivan-suhorukov** [comment]
It's huge, so i put output into txt file:

[mintinstall-output.txt](https://github.com/linuxmint/mintinstall/files/13932705/mintinstall-output.txt)

**Comment by Quantum-mutnauQ** [comment]
Hello,
did you remove the cache before?

**Comment by ivan-suhorukov** [comment]
Ok, now i remove the cache:

[mintinstall-log.txt](https://github.com/linuxmint/mintinstall/files/13988912/mintinstall-log.txt)

**Comment by Quantum-mutnauQ** [comment]
Hello,
Did you upgrade to 21.3 Virginia?

**Comment by ivan-suhorukov** [comment]
Yes, i did the upgrade. I does not make any change.

**Comment by Quantum-mutnauQ** [comment]
I think a good step to find out if it is a problem with some configs or a system problem is, to make a new user set the language to englisch. Than make shure that every thing is at the normel settings. Than start mintinstall.

**Comment by ivan-suhorukov** [comment]
I done that. No change :( Maybe this is going too hard. I thing this is no big trouble, i can install packages in other ways and this is not common trouble as i see.. so you can close this issue if you want.

**Comment by Quantum-mutnauQ** [comment]
you can try it on a live stick to figger out if it is a hardware problem. How many ram do you have?
